testing/btest/language/redef-same-prefixtable-idx.zeek

```diff
@@ -4,12 +4,8 @@
 
 const my_table: table[subnet] of subnet &redef;
 
-redef my_table[3.0.0.0/8] = 1.0.0.0/8;
```

Comment:
I wonder if this was deprecated by accident, but it has been reported as deprecated for a bit, just not sure what the message ever meant for above case :-)
```
warning in ./my_table.zeek, line 3: initialization not preceded by =/+=/-= is deprecated (3.0.0.0/8 = 1.0.0.0/8)
```

Reply:
Vern explained that this is indeed supposed to go. Resolving.